src/firebase/config.js

```diff
@@ -0,0 +1,20 @@
+import firebase from 'firebase';
+import '@firebase/auth';
+import '@firebase/firestore';
+
+const firebaseConfig = {
```

Comment:
All this configuration is okay to be here? Often the credentials for things that are not public matter needs to use environment variables. Are you sure the documentation doesn't suggest something around environment variables or does it suggest committing this to git?

Reply:
None of the documentation that i check say something about committing or not the values to a remote repo, nor does it mention environment variables; that being said, it make sense to use them for this project. I'll make sure to add a .env